packages/components/src/custom-gradient-picker/serializer.js

```diff
@@ -21,7 +21,7 @@ export function serializeGradientColorStop( { type, value, length } ) {
 	return `${ serializeGradientColor( {
 		type,
 		value,
-	} ) } ${ serializeGradientPosition( length ) }`;
+	} ) } ${ length ? serializeGradientPosition( length ) : '' }`;
```

Comment:
Gradient color `length` _(gradient color location - e.g. 50%)_ can be undefined e.g. in `linear-gradient(to top , #3C8067, #FAFBF6)` from the theme called `Spearhead`. On web nothing happens when trying to switch from `linear` to `radial` but on mobile there is a crash since we reuse logic within `serializer.js`. 

Properties of undefined `length`  are then used in function:

```
export function serializeGradientPosition( { type, value } ) {
	return `${ value }${ type }`;
}
```
which makes a crash.

Reply:
To me this seems ok to do, but this code is new to me as well. Maybe @jorgefilipecosta can have a quick look?

Reply:
This change looks good to me 👍 

I guess as an enhancement we can add this logic inside the serializeGradientPosition function directly:
```
export function serializeGradientPosition( position ) {
	if ( ! position ) {
		return '';
	}
	const { value, type } = position;
	return `${ value }${ type }`;
}
```

Reply:
Sure, make sense 👍🏼